golang/example/graph/comment.go

```diff
@@ -256,12 +271,17 @@ func (s *searchCommentInput) QueryMods() []qm.QueryMod {
 // AllComments retrieves Comments based on the provided search parameters
 func (r *Resolver) AllComments(ctx context.Context, args struct {
 	Since    *graphql.ID
-	PageSize int32
+	PageSize *int32
 	Search   *searchCommentInput
 }) (CommentsCollection, error) {
 	result := CommentsCollection{}
+
+	pageSize := 25 // Default page size
```

Comment:
good call on optional `args.PageSize`. though the default `25` should be an unexported var in the package instead of scattering magic number in multiple places

Reply:
👍Thanks!